sda-commons-dependencies/build.gradle

```diff
@@ -37,6 +37,7 @@ dependencies {
   constraints {
     api "com.auth0:java-jwt:3.10.3"
     api "com.github.database-rider:rider-core:1.2.9" // latest version that supports Parameterized tests: https://github.com/database-rider/database-rider/issues/104
+    api "org.postgresql:postgresql:42.2.5"
```

Comment:
This is strange, but I remember that we already had this lately? @dhenneke

Reply:
This seems to be part of the DW2 update: https://github.com/SDA-SE/sda-dropwizard-commons/pull/303/files#diff-c382e801bb689c491e539dc9fc7090f4L51